src/meta/model/src/connection.rs

```diff
@@ -26,6 +26,8 @@ pub struct Model {
     #[sea_orm(primary_key, auto_increment = false)]
     pub connection_id: ConnectionId,
     pub name: String,
+
+    // todo: Private link service has been deprecated, consider using a new field for the connection info
```

Comment:
I think we should deprecate the unnecessary fields in `message PrivateLinkService`, only keeping the necessary fields for `create source`.

Reply:
In long term, the `create connection` statement should be kept, but the semantic of it is simply storing the user-specific `endpoint` and `targets`, *instead of* calling AWS API to create a private link. This aligns with the style of the connection RFC.

Reply:
I am not sure if it is safe to remove the field. The exisiting connection still need this.
After the cloud portal supports building subnets for private link (long ago), we no longer encourage using `create connection` for this case. 

> but the semantic of it is simply storing the user-specific endpoint and targets, instead of calling AWS API to create a private link

IIRC, we store the two fields together with other connector props, rather than in the connection catalog.

Reply:
> IIRC, we store the two fields together with other connector props, rather than in the connection catalog.

I see. Then we may deprecate the entire `message PrivateLinkService`. See also https://stackoverflow.com/a/52787662/5739882.

Reply:
Does the change involve a new migration, cc @yezizp2012 ?